api/github.ts

```diff
@@ -404,29 +406,24 @@ async function repoStats ({topic, owner, repo, ...restArgs}: PathArgs) {
   }
 }
 
-function dependents (type: string) {
+function dependents (type: DependentsType) {
+  const accept = [
+    'text/html',
+    'application/xhtml+xml',
+    'application/xml'
+  ].join(',')
+  const headers = { accept }
```

Comment:
IMO the original one has better readability

```suggestion
  const headers = {
    accept: 'text/html, application/xhtml+xml, application/xml'
  }
```

Reply:
Except original didn't have spaces after commas but nvm I'll return it to its original form. That being said you didn't really sell me _string-~a~rrays_ as a better option but this is your playground so I'm playing by your rules 😄

Reply:
I guess this `accept` header wouldn't change on a foreseeable future, it's more like a static string. In this case, less logic (like `join` method) means less thinking while reading :D